src/components/Home.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
 
-const Home = () => {
+import React from "react";
+import { Button } from "react-bootstrap";
+
+
+const Home = props => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="d-flex flex-column justify-content-center ">
+      <section className="align-self-center text-center">
+        <h1> {"{...}"} </h1>
```

Comment:
I know you are copying the wireframe in the issue, but it's unusual for an app to have a heading of `...`. Instead, let's use the name of the app which is CYF Quiz.

Also, html tag contents should not have spaces at the beginning and end — can we delete them for all tags in this file?

Reply:
Done.